components/namespace-lister/policies/ns-label/.chainsaw-test/chainsaw-test.yaml

```diff
@@ -0,0 +1,104 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/kyverno/chainsaw/main/.schemas/json/test-chainsaw-v1alpha1.json
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: mutate-namespace-konflux
+spec:
+  steps:
+  - name: step-00
```

Comment:
`step-00` isn't a particularly descriptive name - is this required by something, or could we set something more descriptive here?